### PR TITLE
fix(vouchers): update vouchers tab to latest API version [MW-1400]

### PR DESCRIPTION
**Pull request by jorritdenbroeder**
@Aronv90 Would you like to merge? I removed some old stuff, using new API calls now.

### CODE REVIEW

File: src/app/person/show/vouchers/person-show-vouchers.tpl.html [hunk 1]
```diff
@@ -1,39 +1,36 @@
 <h4>Vouchers</h4>
 
-<div class="col-md-4">
-  <!--<button class="btn btn-default" ng-click="createEditVoucher()">Add new voucher</button>-->
-</div>
+<dl class="dl-horizontal">
+  <dt>Credit</dt>
+  <dd>{{ credit.error.message || (debt.value | currency) }}</dd>
```
Comment (Aronv90) [inline src/app/person/show/vouchers/person-show-vouchers.tpl.html]: Moet credit.error.message niet debt.error zijn en debt.value niet credit.value?

File: src/app/person/show/vouchers/person-show-vouchers.tpl.html [hunk 2]
```diff
@@ -1,39 +1,36 @@
...
+
+  <dt>Debt</dt>
+  <dd>{{ debt.error.message || (debt.value | currency) }}</dd>
```
Comment (Aronv90) [inline src/app/person/show/vouchers/person-show-vouchers.tpl.html]: Moet debt.error.message niet debt.error zijn?

File: src/app/person/show/vouchers/person-show-vouchers.tpl.html [hunk 3]
```diff
@@ -1,39 +1,36 @@
...
+
+  <dt>Required value</dt>
+  <dd>{{ requiredValue.error.message || (requiredValue.value | currency) }}</dd>
```
Comment (Aronv90) [inline src/app/person/show/vouchers/person-show-vouchers.tpl.html]: Moet requiredValue.error.message niet requiredValue.error zijn?

Comment (jorritdenbroeder) [inline src/app/person/show/vouchers/person-show-vouchers.tpl.html]: Thanks! Yes, debt.value (right after [dt]Credit[/dt]) should have been credit.value, changed it.

requiredValue is correct by the way. I thought I'd save some time doing it this way ;-) Look at the controller: e.g. credit starts with a value of NULL, after loading it becomes { value: ... } or { error: { message: ... } }. So the value will show only if there's no error

**Pull request merged**